book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(label, author)
+    @title = label
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(book, date)
+    Rental.new(date, self, book)
+  end
```

Comment:
- [ ] We already have the book object, so we don't need to provide it. Instead what we need is the person object and the date.

- [ ] Also, be sure to organize your parameters based on how you defined the constructor in the rental class.

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```

Reply:
Thanks for your help @DammyShittu ! 👍🏾